pkg/assetfetcher/http_test.go

```diff
@@ -850,3 +852,19 @@ func TestNewNexposeSiteAssetsRequestWithExtraSlashes(t *testing.T) {
 
 	assert.Equal(t, "http://localhost/api/3/sites/siteID/assets?page=1&size=100", req.URL.String())
 }
+func TestHasBeenScannedBadTimeField(t *testing.T) {
+	dummyAssetBlankTimeStamp := Asset{History: assetHistoryEvents{AssetHistory{Type: "SCAN", Date: ""}}}
```

Comment:
Suggestion: if you're going to combine multiple test cases in a single test function, you should use [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests) to make it more clear what your test cases are vs your test harness.

This is a good example within this code base: https://github.com/asecurityteam/nexpose-asset-producer/blob/master/pkg/assetfetcher/errors_test.go#L75

Reply:
can do